.github/workflows/release.yaml

```
@@ -9,39 +9,13 @@ env:
-      - name: Build and Test
-        run: ./gradlew clean build
-
   starter-compatibility-tests:
-    name: ${{ matrix.starter }} on Java ${{ matrix.javaVersion }}
-    runs-on: ubuntu-latest
+    name: ${{ matrix.starter }} on Java ${{ matrix.javaVersion }} (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)